package.json

```diff
@@ -1,8 +1,8 @@
 {
   "scripts": {
-    "test": "cross-env ENVIRONMENT=testing jest --watch --verbose",
     "start": "node index.js",
-    "server": "nodemon index.js"
+    "server": "nodemon index.js",
+    "test": "cross-env DB_ENV=testing jest --watch --verbose"
```

Comment:
I believe this should not be changed. We have used ENVIRONMENT rather than DB_ENV throughout the project